en/docs/connectors/catalog/crm-sales/hubspot.crm.commerce.quotes/setup-guide.md

```
@@ -1,103 +1,93 @@
+![Get credentials](/img/connectors/catalog/crm-sales/hubspot.crm.commerce.quotes/setup/clientId_clientSecret.png)
 
-:::tip
-Use a tool like [Postman](https://www.postman.com/) or `curl` to perform the token exchange in step 5.
-:::
+## Step 6: Get the refresh token
 
-## Alternative: Use a private app access token
+1. Construct the authorization URL:
 
-If you prefer not to use OAuth 2.0, you can use a HubSpot Private App:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)